From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pavel Tumakaev <p.tumakaev@omprussia.ru>
Date: Wed, 3 Mar 2021 19:11:52 +0300
Subject: [PATCH] [sailfishos][gecko] Remove static registration of the
 'mozilla/places/History.h' module. Contributes to JB#54255

This fixes the build error: "Exception: Duplicate contract ID:
@mozilla.org/browser/history;1"

Signed-off-by: Pavel Tumakaev <p.tumakaev@omprussia.ru>
---
 toolkit/components/places/components.conf | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/toolkit/components/places/components.conf b/toolkit/components/places/components.conf
index 880f9373f46f..ebdc983215da 100644
--- a/toolkit/components/places/components.conf
+++ b/toolkit/components/places/components.conf
@@ -5,14 +5,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 Classes = [
-    {
-        'cid': '{0937a705-91a6-417a-8292-b22eb10da86c}',
-        'contract_ids': ['@mozilla.org/browser/history;1'],
-        'singleton': True,
-        'type': 'mozilla::places::History',
-        'headers': ['mozilla/places/History.h'],
-        'constructor': 'mozilla::places::History::GetSingleton',
-    },
     {
         'cid': '{e8b8bdb7-c96c-4d82-9c6f-2b3c585ec7ea}',
         'contract_ids': ['@mozilla.org/network/protocol;1?name=moz-anno'],
-- 
2.25.1

